README.rst

```diff
@@ -40,14 +40,59 @@ USAGE
 
 If you want to learn more about coala-bears, its functionality and its usage,
 please take a look at our
-`tutorial <http://coala.readthedocs.org/en/latest/Users/Tutorials/Writing_Bears.html#what-is-a-bear>`__.
+`tutorial <http://coala.readthedocs.org/en/latest/index.html>`__.
 
 ABOUT
 =====
 
-coala-bears is a Python package containing all the bears that is used by coala.
-
-Read more at our `documentation <http://coala.readthedocs.org/en/latest/Users/Tutorials/Writing_Bears.html#guide-to-write-a-bear>`__.
+coala-bears is a Python package containing all the bears that are used by coala.
+It has been split from `coala <https://github.com/coala-analyzer/coala>`_.
+With the initial release, it features 56 bears covering 32 languages.
+You can see all of them `here <https://gist.github.com/Adrianzatreanu/cf2d0c8b2ecd542a4860>`_
+with a brief description each.
+
+coala-bears has bears for famous languages, such as:
```

Comment:
I think this is rather essential information, more like promotional. If we don't convince them right away, that we worth a time,  no one will read the wiki page.